### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by SexyCarrots**
# Ready to GO

### CODE REVIEW

File: docs/source/whitepaper/6-2-pauli-string-expectation_cn.ipynb [hunk 1]
```diff
@@ -0,0 +1,628 @@
...
+   "source": [
+    "## 概述"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "73b70145",
+   "metadata": {},
+   "source": [
+    "我们需要在各种量子算法中评估电路上许多泡利弦项的总和，VQE 中哈密顿量 $H$ 的基态准备就是一个典型的例子。\n",
```
Comment (refraction-ray) [inline docs/source/whitepaper/6-2-pauli-string-expectation_cn.ipynb:24]: 弦 -> 字符串

Comment (refraction-ray) [reply docs/source/whitepaper/6-2-pauli-string-expectation_cn.ipynb:24]: 基态准备 -> 基态制备

File: docs/source/whitepaper/6-2-pauli-string-expectation_cn.ipynb [hunk 2]
```diff
@@ -0,0 +1,628 @@
...
+    "\n",
+    "2. 与 1 类似，但我们通过 ``vmap`` 计算泡利字符串的总和；\n",
+    "\n",
+    "3. $\\langle H \\rangle$ 其中 $H$ 是一个稀疏矩阵；\n",
+    "\n",
+    "4. $\\langle H \\rangle$ 其中$H$ 是一个稠密矩阵；\n",
+    "\n",
+    "5. $H$ 的矩阵乘积算子 (MPO) 的期望值。\n",
+    "\n",
+    "我们以横向场模型（TFIM）为例，其内容为\n",
```
Comment (refraction-ray) [inline docs/source/whitepaper/6-2-pauli-string-expectation_cn.ipynb:37]: 内容 -> 哈密顿量

File: docs/source/whitepaper/6-2-pauli-string-expectation_cn.ipynb [hunk 3]
```diff
@@ -0,0 +1,628 @@
...
+       "<Figure size 1591.6x1288.28 with 1 Axes>"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# 我们使用的电路 ansatz 如下图所示，双量子比特门是梯形布局\n",
```
Comment (refraction-ray) [inline docs/source/whitepaper/6-2-pauli-string-expectation_cn.ipynb:128]: 梯形->阶梯型

File: docs/source/whitepaper/6-2-pauli-string-expectation_cn.ipynb [hunk 4]
```diff
@@ -0,0 +1,628 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "80cc3f92-4d76-470d-b1c0-65c2736c6383",
+   "metadata": {},
+   "source": [
+    "## 主优化循环\n",
+    "\n",
+    "VQE 训练循环（值和 grad 函数不可知）"
```
Comment (refraction-ray) [inline docs/source/whitepaper/6-2-pauli-string-expectation_cn.ipynb:139]: （支持任意梯度函数形式）

File: docs/source/whitepaper/6-2-pauli-string-expectation_cn.ipynb [hunk 5]
```diff
@@ -0,0 +1,628 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0a238264",
+   "metadata": {},
+   "source": [
+    "## 3. 稀疏矩阵\n",
+    "\n",
+    "如果哈密顿量是稀疏的，则可以在空间和时间方面获得显着的计算优势，在这种情况下，算子的稀疏表示是可取的。\n",
+    "这可以通过在两阶段过程中从泡利结构列表转换来以后端不可知的方式提供。\n",
```
Comment (refraction-ray) [inline docs/source/whitepaper/6-2-pauli-string-expectation_cn.ipynb:377]: 不可知 -> 无关
提供 -> 实现

File: docs/source/whitepaper/6-2-pauli-string-expectation_cn.ipynb [hunk 6]
```diff
@@ -0,0 +1,628 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e4d9c11e",
+   "metadata": {},
+   "source": [
+    "## 5. MPO\n",
+    "\n",
+    "TFIM 哈密顿量，作为短程自旋哈密顿量，承认有效的矩阵乘积算子表示。再次强调，这是一个使用 TensorCircuit 的两阶段过程。\n",
```
Comment (refraction-ray) [inline docs/source/whitepaper/6-2-pauli-string-expectation_cn.ipynb:544]: 承认 -> 具有
~~再次强调~~

**Review by refraction-ray — CHANGES_REQUESTED** [review]
Several minor translation issues to be fixed

**Comment by refraction-ray** [comment]
And what is the PR title about 😂

**Comment by SexyCarrots** [comment]
my local repo just turn wild. I had to upload it manually through the GitHub website lol.

**Comment by SexyCarrots** [comment]
# Revised

**Review by refraction-ray — APPROVED** [review]
LGTM

**Pull request merged**